src/js/configs.js

```diff
@@ -157,6 +157,16 @@ module.exports = {
     tls13: '6.0',
     usesOpenssl: true,
   },
+  stunnel: {
+    highlighter: 'ini',
+    latestVersion: '5.56',
+    name: 'stunnel',
+    showSupports: false,
+    supportsHsts: false,
+    supportsOcspStapling: false,
+    tls13: '5.56',
```

Comment:
It sounds like this is either `5.40` like you use in the template or `5.45` as it appears to be in the release notes.